.buildkite/pipeline.build.yml

```diff
@@ -183,9 +183,9 @@
     - bash ./ci/ci.sh prepare_docker
     - ./ci/env/env_info.sh
     - pip install ray[client]
-    - bazel test --config=ci $(./ci/run/bazel_export_options)
-      --test_tag_filters=client_tests,small_size_python_tests
-      -- python/ray/tests/...
+    - bazel test --config=ci $(./ci/run/bazel_export_options) 
+      --test_tag_filters=client_tests,small_size_python_tests 
+      $(bazel query "tests(python/ray/tests/...) intersect attr(tags, civ1, //...)")
```

Comment:
why not just add civ1 into `--test_tag_filters`?

Reply:
I think you want to do:

`$(bazel query "attr(tags, civ1, tests(python/ray/tests/...))")`

you do not want to use `//...` which will parse all the BUILD files, and very likely will fail.

Reply:
> why not just add civ1 into --test_tag_filters

the set of tag is combined with OR, not AND, ugg (https://bazel.build/docs/user-manual#test-tag-filters)